apps/climatemappedafrica/src/payload/globals/HURUMap/PanelOptions.js

```diff
@@ -0,0 +1,57 @@
+import image from "../../fields/image";
+
+const PanelOptions = {
+  label: "Panel Options",
+  fields: [
+    {
+      name: "items",
+      type: "array",
+      label: "Panel Items",
+      fields: [
+        {
+          type: "select",
+          name: "value",
+          label: "Value",
+          options: [
+            { value: "rich-data", label: "Rich Data" },
+            { value: "pin", label: "Pin" },
+          ],
+          localized: true,
+        },
+        image({
+          overrides: {
+            name: "icon",
+            required: true,
+          },
+        }),
+      ],
+    },
+    {
+      type: "row",
+      fields: [
```

Comment:
Not too sure about this...

1. If these are just labels, shouldn't we have a labels group and then these fields inside that
2. I'm assuming these labels are only used in the Panel, right?